### PR TITLE
Add Grunt packages.

### PR DESCRIPTION
**Pull request by sowiecki**
Add Grunt and npm packages.

@wiliajc87

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    uglify: {
+      my_target: {
+        files: {
+          "assets/javascripts/application.min.js": "assets/javascripts/application.js"
+        }
+      }
+    },
+    watch: {
+      files: ["./src/**/*"],
+      tasks: ["browserify", "less"],
```
Comment (thewildpendulum) [inline Gruntfile.js]: Remove `less`

**Comment by sowiecki** [comment]
Done, thanks. @thewildpendulum

**Comment by thewildpendulum** [comment]
@Nase00 :+1:

**Comment by sowiecki** [comment]
@wiliajc87

**Pull request merged**